### ISSUE TITLE
Could use more food options

### ISSUE THREAD

**Issue by merdink** [sibomots/FooFarm#1]
Possibly add Harvestcraft so Hunger Overhaul could have heavier meal options. Highest so far seems to be light meals.

**Comment by sibomots** [comment]
Agreed.   Let's look at this.   The options are the tried and true HarvestCraft and HungerOverhaul.   Let's find out if there's something in 1.12.2 that is novel to examine also.  

Taking suggestions.

**Comment by Drakonas** [comment]
Pam's HarvestCraft would be the best pick, I say. Definite balance of rewards for effort. A lot more light meal options, but take the time and you can have a feast where you eat two and you're good. But it takes a LOT of gathering of tons of ingredients to make those. Hunger Overhaul is a more basic option, but I feel that Pam's HarvestCraft makes everything a bit more exciting. And tbh with a modpack called FooFarm, I think it deserves more than vanilla recipes and HarvestCraft is the most stable and complete I've seen.

**Comment by Drakonas** [comment]
I think I misunderstood Hunger Overhaul. It changes the difficulty of the food intake. I think @merdink was saying that having both might not be a bad idea.

**Comment by SamHarris2020** [comment]
I'm in favor of adding Pam's Harvestcraft while keeping hunger overhaul - and since this isn't a highly technical of mod packs, I think the challenge of trying to automate meals from Harvestcraft would be rather fun.

**Comment by merdink** [comment]
What about Cooking for Blockheads? I enjoy that mod with the kitchen and stuff. I'm just not sure if it would lower the difficulty of the pack with some of its conveniences. I know the pack is supposed to have challenges.

**Comment by SamHarris2020** [comment]
Another interesting mod could be [Spice of Life](https://www.curseforge.com/minecraft/mc-mods/spice-of-life-carrot-edition), which keeps track of how many unique foods a player has eaten and rewards them with new hearts as they pass certain milestone of new foods consumed. Something like that could encourage players to really dive deeply into mods like Pam's Harvestcraft and add motivation to creatively farm.

**Comment by Drakonas** [comment]
[VanillaFoodPantry Mod](https://www.curseforge.com/minecraft/mc-mods/vanillafoodpantry-mod) seems to be a decent option, however it doesn't really add crops. Just extra mob drops and recipes. This would make it ideal to add mid-playthrough, however. And it does seem to offer a decent bit.

![VanillaFoodPantry Foods](https://media.forgecdn.net/attachments/202/466/vfp_foodpoints-mc1.png)

Most likely, this would take some custom configuration for Hunger Overhaul to recognize the recipes' hunger values. Would take some manual modifications per-recipe, most likely. Hunger Overhaul works with Pam's out of the box, I think, but I'm not sure about VanillaFoodPantry Mod.

**Comment by sibomots** [comment]
The issue seems to be resolved initially with the version 1.2 pack.

The list of what was added:
SpiceOfLife-mc1.12-1.3.12.jar
Chameleon-1.12-4.1.3.jar
CTM-MC1.12.2-1.0.2.31.jar
Botania r1.10-363.jar
AgriCraft-2.12.0-1.12.0-a6.jar
jei_1.12.2-4.16.1.301.jar
Pam's Get all the Seeds! 1.12a.jar
Pam's Simple Recipes 1.12.2c.jar
bloodagricraft-1.0.0.jar
StorageDrawers-1.12.2-5.4.2.jar
tinkersjei-1.2.jar
infinitylib-1.12.0.jar
Pam's BrewCraft 1.12.2-1.0.2.jar
CookingForBlockheads_1.12.2-6.5.0.jar
Animus-1.12-2.1.7.jar
pamscookables-1.1.jar
Pam's HarvestCraft 1.12.2zg.jar

**Comment by sibomots** [comment]
Will close issue once more verification is received.

**Comment by Drakonas** [comment]
Things seem to be working together fine. Spice of Life makes it really hard though. Pretty much all vanilla food is useless. But Pam's does seem to make it balance out once you get some good food made. I think ultimately that is the point of the design.

**Comment by merdink** [comment]
Food choices seem good now.
